<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>635</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Scrembled</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>10</horstretch>
     <verstretch>10</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="5">
     <widget class="QPushButton" name="input_accept_button">
      <property name="text">
       <string>Принять</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="scrembler_word_input">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="path_to_file_label">
      <property name="text">
       <string>Путь до файла</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="period_label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Общий период</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLineEdit" name="period_input">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>1024</string>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QPushButton" name="start_button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Закодировать</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="5">
     <widget class="QTableWidget" name="table">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="path_to_file_input"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="input"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="input_label">
      <property name="text">
       <string>Количество сремблеров</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="scrembler_word_label">
      <property name="text">
       <string>Количество бит в ячейке скремблера</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
